pkg/parsers/parsers.go

```diff
@@ -12,17 +12,29 @@ import (
 	"strings"
 )
 
+var (
+	// DefaultHTTPHost Default HTTP Host used if only port is provided to -H flag e.g. docker -d -H tcp://:8080
+	DefaultHTTPHost = "127.0.0.1"
```

Comment:
All things in `pkg/` are meant to be usable by other projects. Here, `pkg/parsers` has a purpose of its own, so I think it's a sane thing to keep it.